zoo/policies/competition_agent.py

```diff
@@ -0,0 +1,29 @@
+from distutils.command.config import config
+import subprocess
+import sys
+import os
+
+from pathlib import Path
+from smarts.core.agent import Agent
+
+
+class CompetitionAgent(Agent):
+    def __init__(self, policy_path):
+        root_path = Path(__file__).absolute().parents[2]
+        req_file = os.path.join(
+            root_path, "competition/track1/submission/requirements.txt"
+        )
+        try:
+            subprocess.check_call(
+                [sys.executable, "-m", "pip", "install", "-r", req_file]
+            )
+        except:
+            print("Failed to install requirement for Competition Agent")
+
+        sys.path.insert(0, policy_path)
```

Comment:
Could you check to see how we might be able to handle multiple `policy.Policy` classes from different submissions? Inserting this path permanently includes the `policy.py` from that submission.

It might involve looking at https://docs.python.org/3.8/library/importlib.html to see if we can load each policy separately.